docs/book/src/topics/ipv6.md

```diff
@@ -0,0 +1,105 @@
+ipv6-0-control-plane-8xqgw   Ready    master   53m   v1.18.8   2001:1234:5678:9abc::4   <none>        Ubuntu 18.04.5 LTS   5.3.0-1034-azure   containerd://1.3.4
+ipv6-0-control-plane-crpvf   Ready    master   49m   v1.18.8   2001:1234:5678:9abc::5   <none>        Ubuntu 18.04.5 LTS   5.3.0-1034-azure   containerd://1.3.4
+ipv6-0-control-plane-nm5v9   Ready    master   46m   v1.18.8   2001:1234:5678:9abc::6   <none>        Ubuntu 18.04.5 LTS   5.3.0-1034-azure   containerd://1.3.4
+ipv6-0-md-0-7k8vm            Ready    <none>   49m   v1.18.8   2001:1234:5678:9abd::5   <none>        Ubuntu 18.04.5 LTS   5.3.0-1034-azure   containerd://1.3.4
+ipv6-0-md-0-mwfpt            Ready    <none>   50m   v1.18.8   2001:1234:5678:9abd::4   <none>        Ubuntu 18.04.5 LTS   5.3.0-1034-azure   containerd://1.3.4
+```
+
+- Nodes have 2 internal IPs, one from each ip family. IPv6 clusters on Azure are run on dual-stack hosts. The IPv6 is the primary IP.
+
+```bash
+kubectl get nodes ipv6-0-md-0-7k8vm -o go-template --template='{{range .status.addresses}}{{printf "%s: %s \n" .type .address}}{{end}}'
+Hostname: ipv6-0-md-0-7k8vm
+InternalIP: 2001:1234:5678:9abd::5
+InternalIP: 10.1.0.5
+```
+
+- Nodes have an IPv6 PodCIDR
+
+```bash
+kubectl get nodes ipv6-0-md-0-7k8vm -o go-template --template='{{.spec.podCIDR}}'
+2001:1234:5678:9a40:200::/72
+```
+
+- Pods have an IPv6 IP
+
+```bash
+kubectl get pods nginx-f89759699-h65lt -o go-template --template='{{.status.podIP}}'
+2001:1234:5678:9a40:200::3
+```
+
+- Able to reach other pods in cluster using IPv6
+
+```bash
+# inside the nginx-pod
+#  # ifconfig eth0
+  eth0      Link encap:Ethernet  HWaddr 3E:DA:12:82:4C:C2
+            inet6 addr: fe80::3cda:12ff:fe82:4cc2/64 Scope:Link
+            inet6 addr: 2001:1234:5678:9a40:100::4/128 Scope:Global
+            UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1
+            RX packets:15 errors:0 dropped:0 overruns:0 frame:0
+            TX packets:20 errors:0 dropped:1 overruns:0 carrier:0
+            collisions:0 txqueuelen:0
+            RX bytes:1562 (1.5 KiB)  TX bytes:1832 (1.7 KiB)
+# ping 2001:1234:5678:9a40::2
+PING 2001:1234:5678:9a40::2 (2001:1234:5678:9a40::2): 56 data bytes
+64 bytes from 2001:1234:5678:9a40::2: seq=0 ttl=62 time=1.690 ms
+64 bytes from 2001:1234:5678:9a40::2: seq=1 ttl=62 time=1.009 ms
+64 bytes from 2001:1234:5678:9a40::2: seq=2 ttl=62 time=1.388 ms
+64 bytes from 2001:1234:5678:9a40::2: seq=3 ttl=62 time=0.925 ms
+```
+
+- Kubernetes services have IPv6 ClusterIP and ExternalIP
+
+```bash
+kubectl get svc
+NAME            TYPE           CLUSTER-IP   EXTERNAL-IP          PORT(S)          AGE
+kubernetes      ClusterIP      fd00::1      <none>               443/TCP          94m
+nginx-service   LoadBalancer   fd00::d2a4   2603:1030:805:1::f   8080:31858/TCP   36s
+```
+
+- Able to reach the workload on IPv6 ExternalIP
+
+```bash
+curl http://\[2603:1030:805:1::f\] -v
```

Comment:
Might want to add a note that you should check if your ISP has ipv6 enabled. I initially ran into this and though was not working. Once connecting from a VM in azure everything worked.

Reply:
Oh yeah good point I actually ran into this too